models/nxos/interfaces/replaced_example_01.txt

```diff
@@ -0,0 +1,15 @@
+
+# Using replaced
+
+- name: Configure mentioned interfaces and replace existing config
+  nxos_interfaces:
+    config:
+      - name: Ethernet1/1
+        description: 'Configured by Ansible'
+        enable: True
+	mtu: 2000
```

Comment:
Minor indentation issue here?

Reply:
Looks like git issue? Weird that it neither shows up on my editor nor if you view the file here including the raw version https://raw.githubusercontent.com/ansible-network/resource_module_builder/f322a357d29267b62df374bd6d2019e01274bc7c/models/nxos/interfaces/replaced_example_01.txt.